sample-app.yaml

```diff
@@ -22,6 +22,15 @@ spec:
               name: ceph-notification-bucket-text
           - configMapRef:
               name: python-ceph-vectordb-text
+        - name: python-ceph-vectordb-image
```

Comment:
create multiple deployment files one for image and another for text with current code